### ISSUE TITLE
Screen reader does not announce visible target line

### ISSUE THREAD

**Issue by becka11y** [fusioncharts/fusioncharts-dist#55]
When navigating this multi series chart with a target  identified, [https://www.fusioncharts.com/extensions-accessibility/mscombi2d.html](https://www.fusioncharts.com/extensions-accessibility/mscombi2d.html), the screen reader never announces the tartget value.  I don't think a fixed target should be announced on each individual bar, though. Perhaps as part of the title or part of the series legend?  
![image](https://user-images.githubusercontent.com/318987/232581769-78b2d5a3-40c0-4852-8caa-07ff9ed61201.png)
